config/clock.rb

```diff
@@ -44,6 +44,10 @@ class Clock
     TeacherTrainingPublicAPI::SyncAllProvidersAndCoursesWorker.perform_async
   end
 
+  every(1.week, 'TeacherTrainingPublicAPI::SyncSubjects') do
```

Comment:
Should the subject sync be part of the wider `SyncAllFromTeacherTrainingPublicAPI`? I think it might be better to sync all TTAPI resources together per hour, than having this run once per week, what do you think?